calculator/main.py

```diff
@@ -58,18 +81,25 @@ def get_history(calc):
         ie:
         ('2016-05-20 12:00:00', 'add', (1, 2), 3),
     """
-    pass
+    records = []
+    for item in calc['history']:
+        records.append(item)
+        
+    return records
 
 
 def reset_history(calc):
     """
     Resets the calculator history back to an empty list.
     """
-    pass
+    calc['history'] = []
 
 
 def repeat_last_operation(calc):
     """
     Returns the result of the last operation executed in the history.
     """
-    pass
+    #First: retrieve info about the last informat
+    if len(calc['history']) > 0:
+        _, operation, params, _ = calc['history'][-1]
+        return perform_operation(calc, operation, params)
```

Comment:
This part of the code is what I understood we were supposed to do: retrieve the info about the most recent operation, run it and return the result. Obviously, the task was different. I must have overlooked it somehow. But I understand all your arguments.